terraform/azure/modules/aks/ssh.tf

```diff
@@ -0,0 +1,24 @@
+resource "random_pet" "ssh_key_name" {
```

Comment:
Will this be recreated on every deployment? I believe that is what happens with random numbers. If not, then good to go

Reply:
I dont see it getting created every time I deploy this.